src/main/java/org/cryptomator/ui/keyloading/hub/HubKeyLoadingModule.java

```diff
@@ -72,6 +72,14 @@ static CompletableFuture<ReceivedKey> provideResult() {
 		return new CompletableFuture<>();
 	}
 
+	@Provides
+	@KeyLoadingScoped
+	@Named("registerException")
+	static AtomicReference<Throwable> provideRegisterException() {
+		return new AtomicReference<>();
+	}
+
+
```

Comment:
What is this used for?

Reply:
I reused the registrationFailed dialog. To differentiate between "Device already exists" and any other generic error, i have to inject the Throwable into the dialog controller. This is the injection object.